### PR TITLE
Adds two new Dionae-themed awayships

### PR DESCRIPTION
**Pull request by Yonnimer**
(third times the charm)
- Adds two new dionae-themed ships, the Rokz and Serz clan ships, essentially rival merchants who specialize in selling products related to space xenofauna and occasionally live specimens. Each ship has three roles, two crewmen and the captain, only the captain is whitelisted.
- Adds some clothing associated with them
- Adds some new Dionae loaodut stuff

First time making awayships like this, not sure if I've messed anything up while doing so.

**Do not merge, plan is to have this added alongside something else**

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

**Review by WickedCybs — APPROVED** [review]

File: maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	
+	l_ear = /obj/item/device/radio/headset/ship
+
+	l_pocket = /obj/item/device/radio
+
+	backpack_contents = list(/obj/item/device/flashlight/lantern = 1, /obj/item/device/flashlight/survival = 1)
+
+/datum/outfit/admin/serz_voidtamer/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(H?.w_uniform)
+		H.w_uniform.color = pick("#6D3175")
```
Comment (Alberyk) [inline maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm:46]: There is no need to have a pick here if there is a single option.

**Review by Alberyk — CHANGES_REQUESTED** [review]

File: maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm (hunk 1 above)
Comment (Yonnimer) [reply maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm:46]: Fixed this

File: maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	uniform = /obj/item/clothing/under/unathi/zozo
+	back = /obj/item/storage/backpack/satchel/hegemony
+	id = null
+	l_pocket = /obj/item/device/radio
+	l_ear = /obj/item/device/radio/headset/ship
+	backpack_contents = list(/obj/item/device/flashlight/lantern = 1, /obj/item/device/flashlight/survival = 1)
+
+/datum/outfit/admin/serz_voidtamer/captain/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(H?.w_uniform)
+		H.w_uniform.color = pick("#6D3175")
```
Comment (NonQueueingMatt) [inline maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm:75]: another bad pick here

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm (hunk 2 above)
Comment (Yonnimer) [reply maps/away/ships/dionae/diona_serz_ship/diona_serz_ship_ghostroles.dm]: Done, double checked both ghostroles for more as well

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**